src/language_server_protocol.rs

```diff
@@ -1652,9 +1652,10 @@ impl LanguageClient {
             return Ok(result);
         }
 
-        self.vim()?
-            .rpcclient
-            .notify("s:FZF", json!([source, NOTIFICATION__FZFSinkCommand]))?;
+        self.vim()?.rpcclient.notify(
+            "s:selectionUI_funcref",
+            json!([source, NOTIFICATION__FZFSinkCommand]),
```

Comment:
`rustfmt` changed indentation here; all I did was to replace `s:FZF` with `s:selectionUI_funcref`.